std/bytes/test.ts

```diff
@@ -124,15 +124,15 @@ Deno.test("[bytes] concat multiple arrays", () => {
   assert(u2 !== joined);
 });
 
-Deno.test("[bytes] contain", () => {
+Deno.test("[bytes] contains", () => {
   const source = encode("deno.land");
   const pattern = encode("deno");
   assert(contains(source, pattern));
 
   assert(contains(new Uint8Array([0, 1, 2, 3]), new Uint8Array([2, 3])));
 });
 
-Deno.test("[io/tuil] copyBytes", function (): void {
```

Comment:
Anyone know why this is [io/util]? Is it on purpose?

Reply:
I rename it to [bytes] given the preceding functions. But let me know if I should revert this.

Reply:
That's okay @yos1p; IIRC this method was in `std/io/util.ts` at some point